src/schedule/zephyr_domain.c

```diff
@@ -56,13 +56,21 @@ static void zephyr_domain_thread_fn(void *p1, void *p2, void *p3)
 {
 	struct zephyr_domain *zephyr_domain = p1;
 	int core = cpu_get_id();
+	uint32_t tick0, tick1, timer_fired;
```

Comment:
Terminology nit: these are zephyr "cycles", not "ticks", which are the coarser unit.

Also: you're using them in a subtraction below (even if it's just to print them), so good form wants these to be signed quantities so the overflow has a natural interpretation (e.g. "2 milliseconds ago" vs. "37 days in the future").  Obviously in practice if Zephyr is doing its job, that subtraction is going to be positive definite.  But you'd be shocked at the number of subtle bugs that can cause time to go backwards.

Reply:
Ack, will fix, I was using the k_uptime_ticks() in the first version and thus the name. And yes, for mergeable version rolloever needs to be handled.